ipa-server-configure-first

```diff
@@ -208,6 +208,14 @@ else
 			fi
 		done
 		cp /etc/volume-version /data/volume-version
+
+		if ! [ -f /etc/ipa/ca.crt ] && [ -f /root/ipa.csr ] ; then
+			cp -p /root/ipa.csr /data/ipa.csr
+			echo "FreeIPA server certificate signing request file is in volume in file ipa.csr."
```

Comment:
I would re-formulate this message as: "FreeIPA CA certificate signing request stored in file ipa.csr located in bind-mounted data volume" or something like this.